internal/pkg/workspace/workspace.go

```diff
@@ -125,6 +125,23 @@ func (ws *Service) writeSummary(projectName string) error {
 	return ws.fsUtils.WriteFile(summaryPath, serializedWorkspaceSummary, 0644)
 }
 
+// AppNames returns the name of all the local applications. For now it
+// extracts the application name from the file name of the corresponding
+// application manifest.
+func (ws *Service) AppNames() ([]string, error) {
```

Comment:
Can we capture this in a GitHub issue with a TODO link here? Otherwise, I'm worried of us forgetting about it :P